libraries/SITL/SIM_Motor.cpp

```diff
@@ -23,26 +23,47 @@ using namespace SITL;
 
 // calculate rotational accel and thrust for a motor
 void Motor::calculate_forces(const struct sitl_input &input,
-                             const float thrust_scale,
                              uint8_t motor_offset,
                              Vector3f &rot_accel,
-                             Vector3f &thrust)
+                             Vector3f &thrust,
+                             const Vector3f &velocity_air_bf,
+                             float air_density,
+                             float velocity_max,
+                             float effective_prop_area,
+                             float voltage)
 {
     // fudge factors
-    const float arm_scale = radians(5000);
     const float yaw_scale = radians(400);
 
-    // get motor speed from 0 to 1
-    float motor_speed = constrain_float((input.servos[motor_offset+servo]-1100)/900.0, 0, 1);
+    const float pwm = input.servos[motor_offset+servo];
+    float command = pwm_to_command(pwm);
+    float voltage_scale = voltage / voltage_max;
```

Comment:
I guess theoretically we could get a divide by zero her in voltage_max is zero..

Reply:
the model would need to have maxVoltage=0, which makes no sense